data/xml/1960.preconf.xml

```diff
@@ -0,0 +1,503 @@
+      <bibkey>nn-1960-questions-discussion-10</bibkey>
+    </paper>
+    <paper id="60">
+      <title>Summation by Chairman</title>
+      <author><first>Willis</first><last>Ware</last></author>
+      <url hash="fde5ebc2">1960.preconf-a.60</url>
+      <bibkey>ware-1960-summation</bibkey>
+    </paper>
+    <paper id="61">
+      <title>Introduction</title>
+      <author><first>Vincent</first><last>Giuliano</last></author>
+      <url hash="6d950c7d">1960.preconf-a.61</url>
+      <bibkey>giuliano-1960-introduction</bibkey>
+    </paper>
+    <paper id="62">
+      <title>The High-Speed General-Purpose Computers in Machine Translation</title>
+      <author><first>B. D.</first><last>Blickstein</last></author>
+      <url hash="83805e79">1960.preconf-a.62</url>
+      <bibkey>blickstein-1960-high</bibkey>
+    </paper>
+    <paper id="63">
+      <title>System Design of a Computer for <fixed-case>R</fixed-case>ussian-<fixed-case>E</fixed-case>nglish Translation</title>
+      <author><first>Robert E.</first><last>Wall</last></author>
+      <url hash="2dfa61b0">1960.preconf-a.63</url>
+      <bibkey>wall-1960-system</bibkey>
+    </paper>
+    <paper id="64">
+      <title>Modern Trends in Character Recognition Systems</title>
+      <author><first>Dimitri A.</first><last>Kellogg</last></author>
+      <url hash="59a38d86">1960.preconf-a.64</url>
+      <bibkey>kellogg-1960-modern</bibkey>
+    </paper>
+    <paper id="65">
+      <title>Questions and Discussion 11</title>
+      <url hash="5c54ff25">1960.preconf-a.65</url>
+      <bibkey>nn-1960-questions-discussion-11</bibkey>
+    </paper>
+    <paper id="66">
+      <title>Summation by Chairman</title>
+      <author><first>Willis</first><last>Ware</last></author>
+      <url hash="3dd6a8d8">1960.preconf-a.66</url>
+      <bibkey>ware-1960-summation-chairman</bibkey>
+    </paper>
+    <paper id="67">
+      <title>Special Report</title>
+      <author><first>Helen</first><last>Bronson</last></author>
+      <url hash="10de25b7">1960.preconf-a.67</url>
+      <bibkey>bronson-1960-special</bibkey>
+    </paper>
+    <paper id="68">
+      <title>Conclusion</title>
+      <author><first>H. P.</first><last>Edmundson</last></author>
+      <url hash="ba8227bf">1960.preconf-a.68</url>
+      <bibkey>edmundson-1960-conclusion</bibkey>
+    </paper>
+    <paper id="69">
+      <title>List of participants</title>
+      <url hash="22282d8c">1960.preconf-a.69</url>
+      <bibkey>nn-1960-list</bibkey>
+    </paper>
+  </volume>
+  <volume id="1" ingest-date="2021-09-29">
+    <meta>
+      <booktitle>Proceedings of the National Symposium on Machine Translation</booktitle>
+      <address>Los Angeles</address>
+      <month>2-5 February</month>
+      <year>1960</year>
+      <editor><first/><last>H.P.EDMUNDSON</last></editor>
+    </meta>
+    <frontmatter>
+      <url hash="614472ec">1960.preconf-1.0</url>
+      <bibkey>pre-1980-1960-national</bibkey>
```

Comment:
This is the full-volume PDF and should go under `<meta>`. We should then break it down and assign individual PDFs to their associated papers.